tox.ini

```diff
@@ -94,8 +95,20 @@ deps =
     pytest
     juju
     psycopg2-binary
-    pytest-operator>0.17.0
+    pytest-operator
     -r{toxinidir}/requirements.txt
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} \
         --durations=0 --asyncio-mode=auto -m legacy_relation
+
+[testenv:bundle-integration]
+description = Run integration tests for legacy relations
```

Comment:
The description should be changed to mention other bundle tests instead of legacy relations.